css/collections-page.styl

```diff
@@ -27,10 +70,16 @@
       .subject-video-controls
         display: initial
 
+      .subject-image-frame
```

Comment:
this is per here: https://github.com/zooniverse/Panoptes-Front-End/blob/c6a6a0e9569adeaeb9a613d358e94e0f235558eb/app/components/frame-viewer.jsx#L51 and I think `image` is intended